src/main/java/org/fundacionjala/at15/katas/bankocr/ValidAccount.java

```diff
@@ -0,0 +1,89 @@
+package org.fundacionjala.at15.katas.bankocr;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ValidAccount {
+
+    private List<String> accountlist;
+    private boolean validation = true;
+    private final int large = 9;
+    private final int valueZero = 0;
+
+    public ValidAccount(String path) {
+        //accountlist = path;
+        accountlist = new ArrayList<>();
+        //added for the test
+        accountlist.add("345882865");
+        accountlist.add("000000051");
+        //changed for the test
+        accountlist.add("49006771x");
+        //changed for the test
+        accountlist.add("12345678x");
```

Comment:
In fact, the way that this `accountslist` is being used is more like a unit test. For the actual implementation code, it should support multiple not-hardcoded cases instead of a limited number of inputs